packages/orm/src/EntityManager.ts

```diff
@@ -2006,79 +2006,124 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW, TX ext
    * - Copy any pending changes (must call .flush() first)
    * - Share any changes made in either EntityManager with the other
    *
+   * @param opts options to control the fork behavior
+   *  - allowPendingChanges - allows the em being forked to have pending changes and forces the resulting em to have
+   *  `mode` set to "in-memory-writes".  Changes to entities will be persisted across ems. Optional. default: false
```

Comment:
"changes to entities will be persisted across ems" makes me think that they'll stay in sync? Like if I fork `em1` into `em2` and then make continual changes in `em1`, I assume they don't show up in `em2`?

Reply:
they don't, ill update the text